### PR TITLE
Centralize documentation about config file location

### PR DESCRIPTION
**Pull request by richsalz**
Addresses comment @dot-asm raised in #2566.

### CODE REVIEW

**Comment by levitte** [comment]
Good start, but needs a bit more work.  I'm not sure that removing *all* info on `OPENSSL_CONF` in the sub-command pages is the way to go.  However, something general that refers back to `openssl(1)` would be suitable:

``` pod
=head1 ENVIRONMENT VARIABLES

=over 4

=item OPENSSL_CONF

Configuration file, see L<openssl(1)>.

= back
```

**Comment by richsalz** [comment]
how about if I put some text in the -config description:
```
Optional; for default value, see L<openssl(1)>
```

**Comment by levitte** [comment]
That works as well.  May I suggest making the link like this:  `L<openssl(1)/COMMAND SUMMARY>`?

**Comment by richsalz** [comment]
done, updated commit pushed.

File: doc/man1/ts.pod [hunk 1]
```diff
@@ -216,9 +216,10 @@ otherwise it is a time stamp token (ContentInfo).
 
 =item B<-config> configfile
 
-The configuration file to use, this option overrides the
-B<OPENSSL_CONF> environment variable. See B<CONFIGURATION FILE
-OPTIONS> for configurable variables. (Optional)
+The configuration file to use.
+Optional; for a description of the default value,
+see L<openssl(1)/COMMAND SUMMARY>.
+See B<CONFIGURATION FILE OPTIONS> for configurable variables.
```
Comment (levitte) [inline doc/man1/ts.pod]: Errrr...  `-config` is documented twice, please remove one of them.

Comment (richsalz) [inline doc/man1/ts.pod]: well, it's defined in two separate sections for each of -query -reply.
I think refactoring out common flags is a good thing to do, but a separate PR.

Comment (levitte) [inline doc/man1/ts.pod]: Ah, didn't notice that...

**Comment by richsalz** [comment]
e9681f8 on master, 31041c4 on 1.1.0

**Pull request closed without merging**